librocksdb_sys/crocksdb/c.cc

```diff
@@ -213,7 +213,16 @@ struct crocksdb_seqfile_t         { SequentialFile*   rep; };
 struct crocksdb_randomfile_t      { RandomAccessFile* rep; };
 struct crocksdb_writablefile_t    { WritableFile*     rep; };
 struct crocksdb_filelock_t        { FileLock*         rep; };
-struct crocksdb_logger_t          { shared_ptr<Logger>  rep; };
+struct crocksdb_logger_t : public Logger{
+  void* rep;
+
+  void (*destructor_)(void*);
+  void (*logv_internal)(void* logger,const char* format, va_list ap);
+
+  virtual void Logv(const char* format, va_list ap) {
```

Comment:
nit: `virtual` keyword is not needed. `override` keyword is needed, i.e:
```
void Logv(const char* format, va_list ap) override {
```

Reply:
Done